overlord/snapstate/snapstate.go

```diff
@@ -285,3 +285,27 @@ func Set(s *state.State, name string, snapst *SnapState) {
 	snaps[name] = &raw
 	s.Set("snaps", snaps)
 }
+
+// All returns information about all active snaps.
+func All(s *state.State) ([]*snap.Info, error) {
```

Comment:
Sure. As we settle all the other changes we can look at how things are being built and perhaps just make All and base ActiveInfos on top.